exercises/C-comparison-operators/exercise.js

```diff
@@ -5,16 +5,16 @@
   The variables should have values that match the expected results.
 */
 
-var studentCount = 16;
-var mentorCount = 9;
-var moreStudentsThanMentors; // finish this statement
+const studentCount = 16;
+const mentorCount = 9;
+const moreStudentsThanMentors = true; // finish this statement
```

Comment:
Perhaps try using the variables on lines 8 and 9 instead of 'true' by using comparison operators.